### PR TITLE
Issue#16 AB - Swipe cards and tailwindcss introduced

### PR DESCRIPTION
**Pull request by ABarranco95**
## Changes
_List Changes Introduced by this PR_
1. Installed Tailwind CSS dependencies for easier styling to the home layout
2. Created swiping cards functionality with dummy data for easy testing. (Firebase data next)

## Purpose
Created cards that represent a user, that can be swiped. Styles with tailwindcss

## Pre-Testing TODOs
run 'npm i ' to install new packages.

### CODE REVIEW

**Comment by kmausolf** [comment]
Judging by the large diff here in the package-lock, we need to ensure everyone on this team is using the same version of node. @ABarranco95  , can you determine what node version your team wants to use? After you've done that, switch to that version in nvm, then run npm install again. That's what's causing that large diff, the package lock is trying to be compatible with too many node versions.

**Review by jjonb — APPROVED** [review]
Looks good to go.

File: client/screens/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  {
+    username: "Colton",
+    location: "Dallas, TX",
+    bio: "Semi-Pro Call of Duty League Player looking for 2 more players to join our org.",
+    photoUrl:
+      "https://img.freepik.com/premium-vector/dark-gamer-cloak-mascot-esport-logo-design_139366-592.jpg?w=2000",
+    id: 3,
+  },
+];
+
```
Comment (kmausolf) [inline client/screens/HomePage.js:39]: As a general housekeeping thing, I'd move this out to another file and import it to keep this file clean.

**Review by kmausolf — APPROVED** [review]
Great work on this!  I learned a bunch about Tailwind!

**Pull request merged**